variables.tf

```diff
@@ -8,6 +8,11 @@ variable "account_name_regex" {
   description = "The Terraform regular expression matching the name of the account(s) that you want to configure (e.g. \"^[[:alnum:]]-production$\").  See [https://www.terraform.io/language/functions/regex] for details on Terraform regular expression syntax."
 }
 
+variable "account_quota_profile" {
+  type        = string
+  description = "The name of the AWS profile (typically found in your .aws/credentials file) whose role has permissions to manage service quotas for the account to configure.  For an example of a role like this, look at TBD."
```

Comment:
Is the `TBD` meant to remain?

Reply:
Good catch!  I meant to go back and update that.  See https://github.com/cisagov/cool-configure-aws-account/pull/3/commits/20f55e1d93a377b925a573e7149be3b371244665 for details.